src/index.js

```diff
@@ -71,3 +71,12 @@ exports.marshalPrivateKey = (key, type) => {
 
   return key.bytes
 }
+
+exports.randomBytes = (number) => {
+  if (!number || typeof number !== 'number') {
```

Comment:
You are not checking if it positive.
<sup>but it might be not needed. /me has no idea how JS behaves</sup>